source/getting-started/core-concepts.md

```diff
@@ -1,58 +1,120 @@
 
-Models represent _persistent state_. For example, a blog application would want
-to save the content of a blog post when a user publishes it, and so the blog
-post would have a model defining it, perhaps called the `Post` model. A model
-typically persists information to a server, although models can be configured to
-save to anywhere else, such as the browser's Local Storage.
+```app/templates/components/my-component.hbs
+<div class="inner">
+  {{yield}}
+</div>
+```
 
-## Routes
+```handlebars
+<span>Hello</span>
+<div class="outer">
+  {{#my-component}}
+    Hello there!
+  {{/my-component}}
+</div>
+```
 
-Routes load a controller and a template. They can also load one or more models
-to provide data to the controller that can then be displayed by the template.
-For example, an `all-posts` route might load all the blog posts from the `Post`
-model, load the `all-posts` controller, and render the `all-posts` template.
-Similarly, a `view-post` route might load the model for the blog post to be
-shown, load the `view-post` controller, and render the `view-post` template.
+Because components deal with DOM, they are also responsible for maintaining
+the bulk of an application's state. Additionally they handle events like
+clicks, keyUp, and other more semtantic actions.
 
 ## The Router
 
-The router maps a URL to a route. For example, when a user visits the `/posts`
-URL, the router might load the `all-posts` route. The router can also load
-nested routes. For example, if our blogging app had a list of blog posts on the
-left of the screen and then showed the current blog post on the right, we'd say
-that the `view-post` route was nested inside the `all-posts` route.
+The router maps a URL to a template. For example, when a user visits the `/posts`
+URL, a configured router may load the `posts` route and template. The router can also load
+nested routes. For example, if a blogging app had a list of blog posts on the
+left of the screen and showed the current blog post on the right, we'd say
+that the `posts/show` route was nested inside the `posts` route.
+
+Perhaps the most important thing to remember about Ember is that **the URL drives
+the application state**. URLs determine what template and route to load, and
+that in turn determines what model must be fetched.
+
+## Models
+
+Models represent persistent state. For example, a blog application would
+persist the content of a blog post when a user publishes it. A blog post model
+would represent the state of that persisted content (such as the post itself,
+the title, and the author).
+
+A model
+typically persists information to a server, although models can be configured to
+save to anywhere. For example to the localStorage API, or IndexedDB.
+
+When visiting a URL, Ember attempts to be smart about what models should be
+loaded. For example, if you visit the `/posts` URL an index of all existing
+`Post` models will be fetched.
```

Comment:
Correct! Only for things ending in `_id` of course :-p https://github.com/emberjs/ember.js/blob/6bb0cc11519aa00c10cb6f086becb7beb75927d3/packages/ember-routing/lib/system/route.js#L1395